src/collective/collectionfilter/interfaces.py

```diff
@@ -42,22 +41,23 @@ class ICollectionFilterBaseSchema(Interface):
     )
 
     target_collection = schema.Choice(
-        title=_(u'label_target_collection', default=u'Target Collection'),
+        title=_(u'label_target_collection', default=u'Alternative Target Collection'),
         description=_(
             u'help_target_collection',
-            default=u'The collection, which is the source for the filter '
-                    u'items and where the filter is applied.'
+            default=u'As an alternative you can select a collection, which is the source for the filter '
+                    u'items and where the filter is applied. If not given, we use the current context as collection.'
         ),
-        required=True,
+        required=False,
         vocabulary='plone.app.vocabularies.Catalog',
-        defaultFactory=utils.target_collection_default,
```

Comment:
Do we need to remove the defaultFactory in order for the feature to work? Could still be handy.
Besides - didn't this defaultFactory alredy cover your use case - not needing to define the target_collection if you're adding the collection filter alredy in the context of a collection?

Reply:
the defaultFactory looked in the aq_chain if there might be a collection in our (parent) context and saved its UID. But that's exactly the problem when copying/pasting content with filter tiles/portlets on it's context. Then you always have to edit all the assigned tiles and select a new collection.

I'd go with the opt-in solution from @agitator to use the (possible) context collection by leaving it empty and optionally select a different collection somwhere else ... with all its downsides ...